doc/getting_started/installation_linux.rst

```diff
@@ -55,7 +55,8 @@ Install the required packages in a Ubuntu host system with:
 .. code-block:: console
 
    $ sudo apt-get install git make gcc g++ python3-ply ncurses-dev \
-	 python3-yaml dfu-util device-tree-compiler
+	 python3-yaml dfu-util device-tree-compiler \
+	 python3-pyelftools
```

Comment:
i don't know what version this installs.
if it's not 0.24, recommend install via pip instead.